examples/reference.yml

```diff
@@ -15,6 +15,10 @@ server:  # Server settings.
   # TLS certificate and key.
   certificate: "/path/to/server.pem"
   key: "/path/to/server.key"
+  # Alternately, you may use LetsEncrypt
```

Comment:
also, for those not familiar with letsencrypt, give a link to the documentation.
the package doc (https://godoc.org/rsc.io/letsencrypt) gives a nice overview